Incomplete refundAddressTransaction

I refunded an order and the resulting `refundAddressTransaction` (obtained via `ob/order/<order-id>`) are incomplete on both the buyer's and vendor's side (although different levels of incompleteness):

Here is the vendor's:

```
    "refundAddressTransaction": {
        "txid": "17821e5bd178cbf255b9f15626eb570a2f0470a845d0f4148e667e5e9e54d539",
        "value": 0,
        "confirmations": 3,
        "height": 1724326,
        "timestamp": "2019-10-23T20:08:48.222222Z",
        "bigValue": "181791"
    }
```

- The main issue is that a currency definition is missing.

Here is the buyer's:

```
    "refundAddressTransaction": {
        "txid": "",
        "value": 0,
        "confirmations": 0,
        "height": 0,
        "bigValue": ""
    }
```

- the `txid` is empty
- the `confirmations` do not match the vendors
- the correct `height` is missing
- the `timestamp` is missing
- the `bigValue` is not set

I'm seeing some of these on the buyer's side - not sure if it's related:

`14:06:57.248 [ERROR] [db/GetByOrderId] unexpected end of JSON input`

*Edit:* After waiting a bit and refreshing, the missing fields mostly come. What still never comes is the currency definition... which is a bit odd, because I have another order that was canceled by the vendor and the currency definition comes after the cancel:

```
    "refundAddressTransaction": {
        "txid": "56d8ffcbcbccd428c5248062d3b595724a44e32722650dc6645702fbed869cb7",
        "value": 0,
        "confirmations": 676,
        "height": 1723654,
        "timestamp": "2019-10-22T15:29:15.683059Z",
        "currency": {
            "code": "LTC",
            "divisibility": 8
        },
        "bigValue": "--662924"
    }
```

Is it possible that "Refund" and "Cancel" create the `refundAddressTransaction` differently?

--- a/core/refunds.go
+++ b/core/refunds.go
@@ -104,6 +104,7 @@ func (n *OpenBazaarNode) RefundOrder(contract *pb.RicardianContract, records []*
 		txinfo := new(pb.Refund_TransactionInfo)
 		txinfo.Txid = txid.String()
 		txinfo.BigValue = outValue.String()
+		txinfo.ValueCurrency = contract.BuyerOrder.Payment.AmountCurrency
 		refundMsg.RefundTransaction = txinfo
 	}
 	contract.Refund = refundMsg

